_posts/2024-06-19-optimizing-opensearch-with-fp16-quantization.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: post
+title:  "Optimizing OpenSearch with Faiss FP16 scalar quantization: Enhancing memory efficiency and cost-effectiveness"
+authors:
+  - naveen
+  - vamshin
+  - tal
+date: 2024-06-19 00:00:00 -0700
```

Comment:
Please update the publish date to 2024-06-27

Reply:
done